.pre-commit-config.yaml

```diff
@@ -15,16 +15,16 @@ repos:
   - id: check-executables-have-shebangs
   - id: check-toml
 
-- repo: https://github.com/psf/black
-  rev: 23.1.0
+- repo: https://github.com/pre-commit/mirrors-autopep8
+  rev: v2.0.2
   hooks:
-  - id: black
+  - id: autopep8
+    args: [--in-place, --recursive, astro_plasma/]
 
 - repo: https://github.com/charliermarsh/ruff-pre-commit
   rev: v0.0.255
   hooks:
   - id: ruff
-    args: [--fix]
```

Comment:
Why do we not want ruff to make fixes?

Reply:
Because it was not commit the code after fixing it on the pre-commit.ci. If you want I can re-enable it. The service allows autofix commit

> autofix pushing was disabled in the configuration. to autofix this pr, comment "pre-commit.ci autofix" on the pull request